src/react-guides/Guides.tsx

```diff
@@ -200,10 +200,10 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
     const { className } = this.props as Required<GuidesProps>;
     return (
       this.props.displayDragPos && (
-        <div 
-          className={`${prefix('wrapper-pos')} ${DISPLAY_DRAG} ${prefix(className)}`} 
-          ref={ref(this, 'displayElement')} 
-          style={this.props.dragGuideStyle} 
+        <div
+          className={`${prefix('wrapper-pos', className)}`}
```

Comment:
You can write like this.
In such case you can delete extra variable classname and use it like props directly
```suggestion
          className={`${prefix('wrapper-pos', this.props.className)}`}
```